Source/WebCore/rendering/PathOperation.cpp

```diff
@@ -50,6 +52,16 @@ const SVGElement* ReferencePathOperation::element() const
     return m_element.get();
 }
 
+const std::optional<Path> ReferencePathOperation::getPath(FloatRect reference, FloatPoint anchor, OffsetRotation rotation) const
+{
+    UNUSED_PARAM(anchor);
```

Comment:
Instead of using `UNUSED_PARAM()` you can just remove the parameter name on line 55.

Reply:
Fixed.